src/signals/incident/definitions/wizard-step-2-vulaan/overlast-van-dieren.ts

```diff
@@ -308,7 +334,7 @@ const overlastVanDieren = {
         extra_dieren_waar_ratten: 'ander_gebouw',
       },
       value:
-        'Het bedrijf, kantoor of ziekenhuis moet zelf een specialist dierplaagbestrijding inhuren. U vindt adressen van specialisten dierplaagbestrijding op [nvbp.org](https://www.nvpb.org/) of [platformplaagdierbeheersing.nl](https://www.platformplaagdierbeheersing.nl). \\\n U kunt dit formulier niet meer verder invullen.',
+        'De eigenaar, woningcorporatie of VVE van de woning kan u helpen de overlast te verminderen. U vindt adressen van specialisten dierplaagbeheersing op [nvbp.org](https://www.nvpb.org/) of [platformplaagdierbeheersing.nl](https://www.platformplaagdierbeheersing.nl). Bekijk ook de [tips om overlast te verminderen](https://www.ggd.amsterdam.nl/ratten-overlast-plaagdieren/ratten/). \\\n U kunt dit formulier niet meer verder invullen.',
```

Comment:
I don't think the first sentence should be changed.

Reply:
Reverted the change